docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3"
+
+services:
+  postgres:
+    environment:
+      - POSTGRES_USER=metaci
+      - POSTGRES_DB=metaci
+      - POSTGRES_PASSWORD=
+    image: postgres:10.6
+    ports: 
+    - "5432:5432/tcp"
+    restart: always
```

Comment:
Please add a volume so the directory where the data is stored is mounted from the host: https://github.com/SFDO-Tooling/MetaShare/blob/master/docker-compose.yml#L37